lib/src/signed_video_internal.h

```diff
@@ -190,11 +190,13 @@ struct _signed_video_t {
 
   signed_video_authenticity_t *authenticity;  // Pointer to the authenticity report of which results
   // will be written.
+  uint8_t received_gop_hash[MAX_HASH_SIZE];  // Received hash list after decoding SEI data while
+  // authenticating. |received_gop_hash| will be compared against |hash_list|.
```

Comment:
The comment says that the `received_gop_hash` will be compared against `hash_list`, but that cannot be feasible since they have different sizes. One has the size of a single hash and the other has a size of multiple hashes.